src/open_inwoner/openzaak/documents.py

```diff
@@ -87,3 +94,37 @@ def download_document(url: str) -> Optional[Response]:
         logger.exception("exception while making request", exc_info=e)
     else:
         return res
+
+
+def upload_document(
+    file: InMemoryUploadedFile, title: str, user_choice: int, source_organization: str
+) -> dict:
+
+    client = build_client("document")
+    if client is None:
+        return
+
+    document_body = {
+        "bronorganisatie": source_organization,
+        "creatiedatum": date.today().strftime("%Y-%m-%d"),
+        "titel": title,
+        "auteur": "Open Inwoner Platform",
+        "inhoud": base64.b64encode(file.read()).decode("utf-8"),
+        "bestandsomvang": file.size,
+        "bestandsnaam": file.name,
+        "taal": "nld",
```

Comment:
I think we're missing status="definitief" here.

Reply:
I thought that the status is defined by another source (definitief - (Definitief) Informatieobject door bevoegd iets of iemand vastgesteld dan wel ontvangen). I will update this with the status field as well.

Reply:
Looking at this again I think you can leave it out, as long as it's set on the InformatieObject in OpenZaak (check the admin) then it's okay. Setting the status would then only be required if it's a document which is not yet definitive, which isn't applicable for us.